pom.xml

```diff
@@ -104,6 +104,14 @@
             <scope>system</scope>
             <systemPath>${project.basedir}/lib/LibsDisguises-9.0.9.jar</systemPath>
         </dependency>
+
```

Comment:
doesn't exist since CoreProtect is closed source. the only downloads they give are the jars on dev.bukkit.org and spigotmc.